mosec/coordinator.py

```diff
@@ -60,6 +80,7 @@ def __init__(
         stage_id: int,
         worker_id: int,
         ipc_wrapper: Optional[Callable[..., IPCWrapper]],
+        timeout: int,
```

Comment:
```suggestion
        timeout: float,
```

Reply:
signal.alarm requires an int parameter  `__seconds`.
It would be better if types of the parameters are consistent.

Reply:
Then there should be an integer check in the `append_worker`.

Reply:
in `_validate_arguments`, I've added `validate_int_ge(timeout, "forward timeout", 0)`.